config/locales/assessor_interface.en.yml

```diff
@@ -13,3 +13,13 @@ en:
             professional_standing: Check professional standing
             first_assessment: First assessment
             second_assessment: Second assessment
+    timeline_events:
+      index:
+        title:
+          assessor_assigned: Assessor assigned
+          reviewer_assigned: Reviewer assigned
+          state_changed: Status changed
+        description:
+          assessor_assigned: Assessor assigned to %{assignee_name}.
```

Comment:
I don't think this reads very well. Maybe we could check with Wes but the user is being assigned as assessor|reviewer to the application.

Reply:
Good point, I left these as a kind of placeholder until the real content/design for this page is done but I'll check with Wes.

Reply:
Wes has suggested an alternative, I've amended the commit!

Reply:
Poetry 👏